Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -277,134 +319,118 @@ Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
-   to be considered.
+   XXX(remove RRSIG)XXX
 
-   Currently IANA SOA SERIAL number for root zone is in the form of
-   YYYYMMDDNN, like 2015111801.  In Yeti root system, IANA SOA SERIAL is
-   directly copied in to Yeti SOA SERIAL.  So once the IANA root zone
-   has changed with a new SOA SERIAL, a new version of the Yeti root
-   zone is generated with the same SOA SERIAL.
+   XXX(remove apex NS RRSet)XXX
 
-   There is a case of Yeti DM operation that when a new Yeti root server
-   added, DM operators change the Yeti root zone without change the SOA
-   SERIAL which introduces inconsistency of Yeti root system.  To avoid
-   inconsistency, the DMs publish changes only when new IANA SOA SERIAL
-   is observed.
+   XXX(add apex Yeti NS RRSet)XXX
 
-   An analysis of IANA convention shows IANA SOA SERIAL change twice a
-   day (NN=00, 01).  Since October 2007 the maximum of NN was 03 while
-   NN=2 was observed in 13 times.
+   XXX(remove apex DNSKEY RRSet)XXX
 
-3.1.2.  Timing of Root Zone Fetch
+   XXX(add apex Yeti DNSKEY RRSet)XXX
 
-   Yeti root system operators do not receive notify messages when IANA
-   root zone is updated.  So each Yeti DM checks the root zone serial
-   periodically.  At the time of writing, each Yeti DM checks to see if
-   the IANA root zone has changed hourly, on the following schedule:
+   XXX(sign zone using specified keys)XXX
 
-                         +-------------+---------+
-                         | DM Operator | Time    |
-                         +-------------+---------+
-                         | BII         | hour+00 |
-                         | WIDE        | hour+20 |
-                         | TISF        | hour+40 |
-                         +-------------+---------+
+   The transformation of Root Zone to Yeti-Root zone takes place
+   independently on each of the Yeti DMs.
 
+   Note SOA.SERIAL preservation during transformation process.
 
+3.3.  Yeti-Root Zone Distribution
 
-Song, et al.            Expires December 11, 2017               [Page 7]
-
-Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
+   XXX(need details about NOTIFY configuration on Yeti-DMs)XXX
 
+   XXX(need details about TSIG authentication on Yeti-DMs)XXX
 
-   Note that Yeti DMs can check IANA root zone more frequently (every
-   minute for example).  A test done by Yeti participant shows that the
-   delay of IANA root zone update from the first IANA root server to
-   last one is around 20 minute.  Once a Yeti DM fetch the new root
-   zone, it will notify all the Yeti root servers with a new SOA serial
-   number.  So normally Yeti root server will be notified in less than
-   20 minute after new IANA root zone generated.  Ideally, if an IANA DM
-   notifies the Yeti DMs, Yeti root zone will be updated in more timely
-   manner.
+   XXX(examples of configuration from DMs and Yeti-Root servers might be
+   handy)XXX
 
-3.1.3.  Information Synchronization
+3.4.  Information Synchronization
```

Comment:
We have a Yeti DM Synchronization document : https://github.com/BII-Lab/Yeti-Project/blob/master/doc/Yeti-DM-Sync.md

And we have another Synchronization document during MZSk based on normal Synchronization: https://github.com/BII-Lab/Yeti-Project/blob/master/doc/Yeti-DM-Sync-MZSK.md